tests/cases/compiler/jsdocPropertyAccessWithType.ts

```diff
@@ -0,0 +1,9 @@
+// @allowJs: true
```

Comment:
Can you please 1. Rename to jsdocPrototypePropertyAccessWithType 2. Move to conformance/jsdoc